clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -193,6 +193,13 @@ New checks
   replacing with ``starts_with`` when the method exists in the class. Notably,
   this will work with ``std::string`` and ``std::string_view``.
 
+- New :doc:`modernize-use-std-numbers
+  <clang-tidy/checks/modernize/use-std-numbers>` check.
+
+  Finds constants and function calls to math functions that can be replaced
+  with c++20's mathematical constants from the ``numbers`` header and
```

Comment:
`C++20`.

Reply:
Do you just want the `C` to be big, or do you want the version inside of single or double `?

Reply:
Uppercase `C`. See https://en.cppreference.com.

Reply:
Thx, done